distrho/DistrhoPlugin.hpp

```diff
@@ -581,24 +581,44 @@ struct TimePosition {
           Default constructor for a null BBT time position.
         */
         BarBeatTick() noexcept
-            : valid(false),
-              bar(0),
-              beat(0),
-              tick(0),
-              barStartTick(0.0),
-              beatsPerBar(0.0f),
-              beatType(0.0f),
-              ticksPerBeat(0.0),
-              beatsPerMinute(0.0) {}
+        {
+            clear();
+        }
+
+       /**
+          Reinitialize this position using the default null initialization.
+        */
+        void clear() noexcept
+        {
+            valid = false;
+            bar = 0;
+            beat = 0;
+            tick = 0;
+            barStartTick = 0.0;
+            beatsPerBar = 0.0f;
+            beatType = 0.0f;
+            ticksPerBeat = 0.0;
+            beatsPerMinute = 0.0;
+        }
     } bbt;
 
    /**
       Default constructor for a time position.
     */
     TimePosition() noexcept
-        : playing(false),
-          frame(0),
-          bbt() {}
+    {
```

Comment:
it is a bit duplicated code, but the values should be initialized in the constructor, it is good practice and we can get warnings if we don't do that.

I am not a native English speaker, I am not sure from clear vs reset which one makes more sense..
So we can have it as "clear".